pxf/pxf-jdbc/src/test/java/org/apache/hawq/pxf/plugins/jdbc/JdbcPartitionFragmenterTest.java

```diff
@@ -164,8 +163,8 @@ public void inValidParameterFormat() throws Exception {
         when(inputData.getUserProperty("RANGE")).thenReturn("100:200");
         try {
             new JdbcPartitionFragmenter(inputData);
-            fail("Expected an ArrayIndexOutOfBoundsException");
-        } catch (ArrayIndexOutOfBoundsException ex) {
+            fail("Expected an UserDataException");

```

Comment:
use @Test(expected = UserDataException.class)  so that there is no need for try / catch / fail lines : https://github.com/junit-team/junit4/wiki/exception-testing